crates/open_ai/src/open_ai.rs

```diff
@@ -445,15 +445,11 @@ pub async fn stream_completion(
 
         match serde_json::from_str::<OpenAiResponse>(&body) {
             Ok(response) if !response.error.message.is_empty() => Err(anyhow!(
-                "Failed to connect to OpenAI API: {}",
+                "Failed to connect to API: {}",
```

Comment:
I think logging the `api_url` here would make it less confusing

Reply:
Done